.github/workflows/procedural.yaml

```diff
@@ -40,8 +40,65 @@ jobs:
     name: Waiting for Engineering
     runs-on: ubuntu-latest
     if: github.event_name == 'issue_comment' && !github.event.issue.pull_request
-      && contains(github.event.issue.labels.*.name, 'waiting-for-author')
     steps:
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install jq
+      - name: Get board column of issue
+        id: extract_board_column
+        continue-on-error: true
+        run: |
+          # Request all issues from project (server-side filtering is not supported by GitHub yet)
+          gh api graphql --paginate -F issue=$ISSUE -F project=$PROJECT -F owner=$OWNER -F repo=$REPO -f query='
+            query board_column($issue: Int!, $project: Int!, $owner: String!, $repo: String!, $endCursor: String) {
+              repository(owner: $owner, name: $repo) {
+                issue(number: $issue) {
+                  projectV2(number: $project) {
+                    items(first: 100, after: $endCursor) {
+                      nodes {
+                        fieldValueByName(name: "Status") {
+                          ... on ProjectV2ItemFieldSingleSelectValue {
+                            name
+                          }
+                        }
+                        content {
+                          ... on Issue {
+                           id
+                           title
+                           number
+                           repository {
+                             name
+                             owner {
+                               login
+                              }
+                            }
+                          }
+                        }
+                      }
+                      pageInfo {
+                         hasNextPage
+                         endCursor
+                      }
+                    }
+                  }
+                }
+              }
+            }
+            ' > api_result
+            # Get board column for issue
+            board_column=$(cat api_result | jq -r ".data.repository.issue.projectV2.items.nodes[] |
+                select (.content.number == $ISSUE and .content.repository.name == \"$REPO\" and .content.repository.owner.login == \"$OWNER\") |
+                .fieldValueByName.name")
```

Comment:
`jq` takes the file as the last argument, so probably `cat` is unnecessary here.

Reply:
Thanks, changed